common/changes/@itwin/core-frontend/extensions-alpha_2022-02-25-18-35.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Move Extension APIs to alpha",
```

Comment:
This shouldn't be needed as the Extension APIs were only just added in this release.

Reply:
deleted

Reply:
i think we still need the change log, just no comment

Reply:
re-added